style.css

```diff
@@ -0,0 +1,93 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3c3a39;
+  height: 55px;
```

Comment:
- [ ] [_OPTIONAL_] by default the body has a margin of 8px which is creating a gap. Please fix that.
- [ ] [_OPTIONAL_] To stick with the design please use the background images provided in the design.

### Your Implementation

![image](https://user-images.githubusercontent.com/59416136/159123883-87a8291a-dc2d-4a64-acea-89f4f2b9c843.png)

### Figma's Design

![image](https://user-images.githubusercontent.com/59416136/159123939-8b54b0d6-6d0e-4add-8096-f44b891d597a.png)

Reply:
@Wes-Isaac, @zainsadaqat  Thanks for the review.